oslo_cache/backends/redis.py

```diff
@@ -0,0 +1,45 @@
+from dogpile.cache.backends import redis as redis_backend
+from oslo_cache import core
+__all__ = [
+    'RedisSentinelTLSBackend'
+]
+class RedisSentinelTLSBackend(redis_backend.RedisSentinelBackend):
+    def __init__(self,arguments):
+        """
+        Wrapper class for dogpile.cache.redis_sentinel.
+        Currently only translates arguments for dogpile.cache.backends.RedisSentinelBackend
+        because user is unable to pass them using oslo config in it's current format
+        arguments:
+           sentinels: list of sentinels formated as sentinel1IP:port,sentinel2IP:port
+           ssl: turns on TLS
+           ssl-<keyfile,certfile,ca_certs>: TLS certificate, key and CA
+           username,password: login credentials 
+        """
+        # Reformat sentinels into (ip <string>, port <int>) as required by dogpile.cahce.redis_sentinel
+        sentinels=[tuple([ int(spl) if i==1 else spl for i,spl in enumerate(ip_port.split(":"))]) for ip_port in arguments['sentinels'].split(',')]
+        # Username and password must be in connection_kwargs dictionary
+        reformated_kwargs = dict(
+                username=arguments['username'],
+                password=arguments['password']
+        )
+        if 'ssl' in arguments and arguments['ssl']:
+            reformated_kwargs.update(
+                ssl=arguments['ssl'],
+                ssl_certfile=arguments['ssl_certfile'],
+                ssl_keyfile=arguments['ssl_keyfile'],
+                ssl_ca_certs=arguments['ssl_ca_certs']
+            )
+        # Both connection and sentinel kwargs are needed 
+        arguments.update(
+            sentinels=sentinels,
+            connection_kwargs=reformated_kwargs,
+            sentinel_kwargs=reformated_kwargs
+        )
+        """
+        Usename and password must be deleted from arguments dict
+        after being added to connection_kwargs, otherwise there
+        will be AUTH error
+        """
+        del arguments['username']
+        del arguments['password']
+        return super(RedisSentinelTLSBackend, self).__init__(arguments)
```

Comment:
```suggestion
        return super(RedisSentinelTLSBackend, self).__init__(arguments)

```

Newline at the end of file.